src/radical/pilot/agent/resource_manager/slurm.py

```diff
@@ -78,7 +78,8 @@ def _configure(self):
         self.cores_per_node = self._cfg.get('cores_per_node', 0)
         self.gpus_per_node  = self._cfg.get('gpus_per_node',  0)  # FIXME GPU
         self.mem_per_node   = self._cfg.get('mem_per_node',   0)
-
+        self._log.debug('lfs path environment value %s', os.environ.get('$LOCAL', None))
```

Comment:
@iparask shouldn't it be `os.environ.get('LOCAL')`? (without `$`, which is not part of the name)

Reply:
Fixed